### PR TITLE
Testing Farm: load FMF from commit_sha of source/fork of PR

### PR DESCRIPTION
**Pull request by jpopelka**
and give better status when `summary` is empty

### CODE REVIEW

File: packit_service/worker/testing_farm.py [hunk 1]
```diff
@@ -66,21 +66,15 @@ def _payload(self, build_id: int, chroot: str) -> dict:
...
+        # url of the fork from which the PR has been created
+        from_url = pr.source_project.get_web_url()
         return {
             "api_key": self.service_config.testing_farm_secret,
             "test": {
                 "fmf": {
-                    "url": url,
-                    "ref": ref,
+                    "url": from_url,
+                    "ref": self.metadata.commit_sha,
```
Comment (lachmanfrantisek) [inline packit_service/worker/testing_farm.py:77]: Looks good to me.

File: packit_service/worker/handlers/testing_farm_handlers.py [hunk 2]
```diff
@@ -84,12 +84,16 @@ def run(self) -> TaskResults:
 
         if self.result == TestingFarmResult.running:
             status = CommitStatus.running
+            summary = self.summary or "Tests are running ..."
```
Comment (lachmanfrantisek) [inline packit_service/worker/handlers/testing_farm_handlers.py:87]: Will they send the summary? Is it some planned feature or just a fallback?

Comment (jpopelka) [reply packit_service/worker/handlers/testing_farm_handlers.py:87]: @thrix said yesterday that there should be something like "1 test passed, 1 failed" in the summary, not sure what's the current status

**Review by lachmanfrantisek — APPROVED** [review]

File: packit_service/worker/handlers/testing_farm_handlers.py (hunk 2 above)
Comment (thrix) [reply packit_service/worker/handlers/testing_farm_handlers.py:87]: yeah, we will, sorry still not done, but I am close ...

**Pull request merged**